README.md

```diff
@@ -2,6 +2,16 @@
 
 [![Build Status](https://travis-ci.com/jackbsteinberg/get-originals-rewriter.svg?branch=master)](https://travis-ci.com/jackbsteinberg/get-originals-rewriter)
 
+## Introduction
+
+When adding built-in modules to the web platform,
```

Comment:
Let's note how this is a Chromium-specific concern. Something like "Chromium is implementing some built-in module proposals in unprivileged JavaScript. To make this possible, we need..."